examples/test_stepreport/stepreport_example.py

```diff
@@ -109,3 +109,61 @@ def test_run(self):
         self.assertAlmostEqual(voltage, 4, delta=1, msg="Check voltage device")
 
         logging.info(f"I HAVE RUN 0.1.1 for tag {self.tag}!")
+
+
+@pykiso.define_test_parameters(
+    suite_id=1,
+    case_id=3,
+)
+class WrapLongTextTest(pykiso.BasicTest):
+    """This test shows wrapping of long results into foldable html elements"""
+
+    def setUp(self):
+        """Set header information and check setup conditions"""
+        super().setUp()
+        # additional data to include in the step-report
+        self.step_report.header["Version_device"] = "2022-1234"
+
+    def test_run(self):
+        """Write long results to the step report to show how foldable html
+        elements get used to make the report overview more readable
+        """
+        logging.info(
+            f"--------------- RUN: {self.test_suite_id}, {self.test_case_id} ---------------"
+        )
+        # since we use big results
+        self.maxDiff = None
```

Comment:
whats this ? Can we remove that ?

Reply:
Removed.
At first I had a bigger diff for the big test results which could not be displayed in the console. this fixed that, but since I made the result size a bit more reasonable it is not needed any more.